src/assets/js/themes.js

```diff
@@ -1,49 +1,48 @@
 /* theme toggle buttons */
-(function() {
-    var enableToggle = function(btn) {
+(function () {
+    const enableToggle = btn => {
```

Comment:
we were using `var` and `iife` (ES5-only) here, as the js won't be compiled? I'm not sure, but it could have a reason. :)

Reply:
converted to ES5